### DISCUSSION TITLE
[开发者公告] PR 分支策略调整

### DISCUSSION THREAD

**Discussion by the1812** [the1812/Bilibili-Evolved#2946] [分享 / Share]
为了让功能修复类的更改能够更快速地应用到正式版中, `preview` 分支已切出两个子分支:

- `preview-fixes` 功能修复, 合并至 `preview` 和 `master`
- `preview-features` 新功能, 只合并至 `preview` (`preview` 仍然会定期合并至 `master`, 只是节奏没有上面那个那么快)

PR 时需要向以上两个分支中选择一个发起, 这个分支策略只影响 PR; 预览版脚本本体及功能的发布, 仍然在 `preview` 分支.
CONTRIBUTING 文档也已更新.

**Comment by FoundTheWOUT** [comment]
这策略是不是有点问题，preview-features 历史和 preview 不一样，导致向 preview 提PR时会携带很多历史信息。我觉得让开发者自己去 hold 这个 features 分支就好了

**Reply by the1812** [reply]
不应该向 preview 提 PR 啊

**Reply by FoundTheWOUT** [reply]
啊，这样的吗，我还要是给 preview 提 PR 呢。打扰了
